components/Record/AnswerTo.tsx

```diff
@@ -1,28 +1,70 @@
-import React from "react";
-import { View, StyleSheet, Pressable, Text, Image } from "react-native";
+import React from 'react';
+import { View, StyleSheet, Pressable, Text, Image } from 'react-native';
 
 interface PropTypes {
-  imageUrl: string;
-  username: string;
-  hashtags: string[];
+    imageUrl: string | undefined;
+    username: string | undefined;
+    hashtags: string[] | undefined;
 }
 
 export const AnswerTo = ({ imageUrl, username, hashtags }: PropTypes) => {
-  const i = {
-    uri: imageUrl,
-  };
-  return (
-    <View
-      style={{
-        justifyContent: "center",
-        alignItems: "center",
-      }}
-    >
-      <View>
-        <Image source={i} />
-        <Text>{username}</Text>
-      </View>
-      <Text style={{ fontWeight: "bold", fontSize: 24 }}>Answer to</Text>
-    </View>
-  );
+    if (imageUrl === undefined) {
+        imageUrl =
+            'https://i.pinimg.com/236x/20/1f/01/201f016bd3a8576fc6cfc872ecac648e--dwight-schrute-hero-.jpg';
+    }
+    const picObject = {
+        uri: imageUrl,
+    };
+    const shouldRender = username !== undefined && hashtags !== undefined;
```

Comment:
den här kommer troligen köras flera gånger. Console logga shouldRender

Reply:
eller maby not såg nu att det var en const function

Reply:
Vet inte riktigt hur du menar 🤔 . Denna är tänkt typ som en "säkerhet" ifall man skapar en
 `<RecordingScreen recordingScreenType={PostType.ANSWER}/>` och inte gav den `answerInfo` proppen.